R/mod-panel-section.R

```diff
@@ -148,6 +148,30 @@ mod_panel_section_server <- function(input, output, session, synapse, syn, user,
     })
   })
   certified <- dccvalidator::check_certified_user(user$ownerId, syn = syn)
+  
+  observeEvent(input$submission, {
+    if (!is.null(input$submission) && nchar(input$submission) > 0) {
+      query <- "SELECT * FROM {submissions_table} WHERE scorer = {user$ownerId} AND form_data_id = {submission_id()}"
+      
+      result <- readr::read_csv(
+        syn$tableQuery(
+          glue::glue(
+            query
+          )
+        )$filepath
+      )
+
+      if (nrow(result) > 0) {
+        updateTextInput(session, "overall_score", value = result$overall_score[1])
```

Comment:
I'm coming at this from a naive angle.  Do we ever expect more than one to be returned?

Reply:
@thomasyu888 No, there is only one overall score per submission (form_data_id).

@erichare We shouldn't filter on `scorer` for queries to the Submissions table; it doesn't matter who added/updated the singular row for a given submission, if it exists we want it.